### PR TITLE
[Fix](nereids)Fix Bug when grouping has same grouping set

### PR DESCRIPTION
**Pull request by feiniaofeiafei**
fix sql like select a from t1 group by grouping sets((a),(a));

### CODE REVIEW

**Comment by feiniaofeiafei** [comment]
run buildall

File: docs/zh-CN/community/design/grouping_sets_design.md [hunk 1]
```diff
@@ -458,7 +458,7 @@ SELECT k1, k2, GROUPING_ID(k1,k2), SUM(k3) FROM t GROUP BY GROUPING SETS ((k1, k
 
 ```
 
-可以看到，其结果与对 GROUPING SETS 子句后每个子集进行 GROUP BY 后再进行 UNION 的结果一致。
+可以看到，其结果与对 GROUPING SETS 子句后每个子集进行 GROUP BY 后再进行 UNION ALL的结果一致。
```
Comment (morrySnow) [inline docs/zh-CN/community/design/grouping_sets_design.md:461]:
```suggestion
可以看到，其结果与对 GROUPING SETS 子句后每个子集进行 GROUP BY 后再进行 UNION ALL 的结果一致。
```

**Review by morrySnow — DISMISSED** [review]

**Review by starocean999 — DISMISSED** [review]

**Comment by Mryange** [comment]
run buildall

**Comment by feiniaofeiafei** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by starocean999 — APPROVED** [review]

**Pull request merged**